src/components/messenger/list/index.tsx

```diff
@@ -138,6 +141,12 @@ export class Container extends React.Component<Properties, State> {
     this.props.fetchRewards({});
   }
 
+  componentDidUpdate(): void {
+    if (this.props.isMessengerFullScreen && !this.props.activeMessengerId) {
+      this.props.openConversation(this.props.conversations[0]?.id);
+    }
+  }
+
```

Comment:
We should manage this in a saga somewhere. Pretty sure there's already a saga for opening fullscreen mode?

Reply:
Handled both flows (login + expand) in saga.